libs/@local/hashql/ast/tests/ui/lowering/name-mangler/onion.jsonc

```diff
@@ -6,7 +6,7 @@
   "U",
   [
     "*",
-    ["fn", { "#tuple": ["T", "T"] }, { "#struct": {} }, "_", ["is", "a", "T"]],
+    ["fn", { "#tuple": ["T", "U"] }, { "#struct": {} }, "_", ["is", "a", "T"]],
```

Comment:
I guess `"T", "T"` would have failed?

Reply:
Correct, that's the new duplicate checking that I added, there's an accompanying test that verifies this.